order.go

```diff
@@ -137,6 +137,25 @@ type OrdersResource struct {
 	Orders []Order `json:"orders"`
 }
 
+type ShippingLine struct {
+	ID                  int              `json:"id"`
+	Title               string           `json:"title"`
+	Price               *decimal.Decimal `json:"price"`
+	Code                string           `json:"code"`
+	Source              string           `json:"source"`
+	Phone               string           `json:"phone"`
+	RequestedFufillment string           `json:"requested_fulfillment_service_id"`
```

Comment:
I would say be verbose

Reply:
I agree, verbose is better. There is also a tiny typo, a missing `l` in in `Fufillment`. I suggest:
`RequestedFulfillmentServiceID`

Reply:
That makes sense. I've changed it